crates/generator/src/generator.rs

```diff
@@ -129,6 +129,9 @@ impl Generator {
             .ok_or(ValidateError::UnknownSUDT)?;
         if sudt_id != CKB_SUDT_ACCOUNT_ID {
             // check SUDT balance
+            if amount <= 0 {
```

Comment:
Remove this, the code below already does the verification.

Reply:
User can't withdrawal 0 SUDT when non-CKB sudt_id exists.